src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -5824,15 +5824,15 @@ private bool IsPossibleName()
             return this.IsTrueIdentifier();
         }
 
-        private ScanTypeFlags ScanType()
+        private ScanTypeFlags ScanType(bool forPattern = false)
```

Comment:
>bool forPattern = false [](start = 39, length = 23)

The parameter is not used.

Reply:
Recorded in https://github.com/dotnet/roslyn/issues/23222